src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +15,13 @@ public class UserService {
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
-        return null;
+        User findUser = null;
+        for (User user : users) {
+            if (user.getEmail().equals(email)) {
+                findUser = user;
+                break;
```

Comment:
more usual approach would be to return user here, and null at the end of the method